Leanplum-SDK/Classes/Managers/Networking/LPRequest.m

```diff
@@ -49,24 +50,34 @@ + (LPRequest *)get:(NSString *)apiMethod params:(NSDictionary *)params
 {
     LPLogType level = [apiMethod isEqualToString:LP_METHOD_LOG] ? LPDebug : LPVerbose;
     LPLog(level, @"Will call API method %@ with arguments %@", apiMethod, params);
+    
+    [[LPCountAggregator sharedAggregator] incrementCount:@"get_lprequest"];
```

Comment:
could you add LPCountAggregator as an instance variable / dependency to this class too?